src/containers/HomeContainers/Home.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState, useEffect } from 'react'
+import { Text, FlatList, View } from 'react-native'
+import { getStarredOfferingsData } from '../../api/offerings'
+import CourseCard from '../../components/Cards/CourseCard'
+import styles from './Home.style'
+
+function fetchCourseInfo(setCourses) {
```

Comment:
Consider moving this function to be inside the `Home` component. Then you won't need to pass in `setCourses`, it will be in scope.